src/components/ReviewList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { getReviews } from "../api";
+
+export const ReviewList = () => {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getReviews().then((reviewList) => {
+      setReviews(reviewList);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) return <p>Loading...</p>;
+
+  return (
+    <section>
+      <h2>Find a Review for all of your favourite tabletop games!</h2>
+      <ul>
+        {reviews.map((review) => {
+          return (
+            <li className="ReviewCard" key={review.review_id}>
```

Comment:
I'm not 100% sure an <li> is the best element to use here, I think a div would be fine